doc/ref_cert/RC1/chapters/chapter03.md

```diff
@@ -237,57 +250,61 @@ According to
-| .\*test_routers_negative.DvrRoutersNegativeTest                                                                 | dvr                                   |
-| .\*test_routers_negative.DvrRoutersNegativeTestExtended                                                         | dvr                                   |
-| .\*test_routers_negative.HaRoutersNegativeTest                                                                  | l3-ha                                 |
-| .\*test_security_groups.RbacSharedSecurityGroupTest                                                             | rbac-security-groups                  |
-| .\*test_subnetpool_prefix_ops                                                                                   | subnetpool-prefix-ops                 |
-| .\*test_subnetpools.SubnetPoolsSearchCriteriaTest.test_list_validation_filters                                  | filter-validation                     |
-| .\*test_subnets.SubnetsSearchCriteriaTest.test_list_validation_filters                                          | filter-validation                     |
-| .\*test_timestamp.TestTimeStamp.test_segment_with_timestamp                                                     | standard-attr-segment                 |
-| .\*test_trunk.TrunkTestInheritJSONBase.test_add_subport                                                         | https://launchpad.net/bugs/1863707    |
-| .\*test_trunk.TrunkTestMtusJSON                                                                                 | vxlan                                 |
-| .\*test_trunk_negative.TrunkTestJSON.test_create_subport_invalid_inherit_network_segmentation_type              | vxlan                                 |
-| .\*test_trunk_negative.TrunkTestMtusJSON                                                                        | vxlan                                 |
-| .\*test_qos.QosMinimumBandwidthRuleTestJSON                                                                     | https://gerrit.opnfv.org/gerrit/69105 |
-| .\*network.test_tags                                                                                            | tag-ext                               |
-| .\*test_routers.RoutersIpV6Test.test_create_router_set_gateway_with_fixed_ip                                    | https://launchpad.net/bugs/1676207    |
-| .\*test_routers.RoutersTest.test_create_router_set_gateway_with_fixed_ip                                        | https://launchpad.net/bugs/1676207    |
-| .\*test_network_basic_ops.TestNetworkBasicOps.test_router_rescheduling                                          | l3_agent_scheduler                    |
-| .\*test_network_advanced_server_ops.TestNetworkAdvancedServerOps.test_server_connectivity_cold_migration_revert | https://launchpad.net/bugs/1836595    |
+| test rejection regular expressions                                                                                    | reasons                               |
```

Comment:
Nitpick -- for easy search
These are largely in alphabetic order except for the entries at the end -- can they be arranged alphabetically; for example, qos and routers moved to their alphabetic position.